README.md

```diff
@@ -32,72 +36,90 @@ You can chain multiple AI models together to create a pipeline before hitting th
 - VectorDB integration
 
 ## Requirements
+
 - OpenAI API key
 - Postgres
 - Redis
 
-
 #### OpenShield is a firewall designed for AI models.
 
-
 ### Endpoints
+
 ```
 /openai/v1/models
 /openai/v1/models/:model
 /openai/v1/chat/completions
 ```
 
 ## Demo mode
-We are generating automatically demo data into the database. You can use the demo data to test the application.
 
+We are generating automatically demo data into the database. You can use the demo data to test the application.
 
 ```shell
 cd demo
 cp .env.example .env
+cp ./example_config ./config.yaml/config
 ```
 
-You need to modify the .env file with your OpenAI API key and HuggingFace API key.
+You need to modify the .env file with your OpenAI API key and HuggingFace API key. Here's how to obtain these keys:
+
+1. OpenAI API key:
+   - Sign up for an OpenAI account at https://platform.openai.com/signup
+   - Once logged in, go to https://platform.openai.com/api-keys
+   - Click on "Create new secret key" to generate your API key
+
+2. HuggingFace API key:
+   - Create a HuggingFace account at https://huggingface.co/join
+   - Go to your account settings: https://huggingface.co/settings/token
+   - Click on "Create new token" to create your API key
+
+After obtaining both keys, update your .env file with the appropriate values.
 
 ```shell
-docker-compose build
-docker-compose up
+docker compose build
```

Comment:
That's incorrect. `docker-compose build`

Reply:
docker-compose is the v1 and soon to be deprecated of compose https://docs.docker.com/compose/migrate/